src/js/stores/MarathonStore.js

```diff
@@ -217,10 +236,17 @@ var MarathonStore = Store.createStore({
       case ActionTypes.REQUEST_MARATHON_GROUPS_SUCCESS:
         MarathonStore.processMarathonGroups(action.data);
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_SUCCESS:
+        MarathonStore.processMarathonDeployments(action.data);
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ERROR:
         MarathonStore.processMarathonGroupsError();
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ERROR:
+        MarathonStore.processMarathonDeploymentsError();
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ONGOING:
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ONGOING:
```

Comment:
@jfurrow the pattern that exists on the application is what @mlunoe said, we have a unique handler for the ONGOING requests, we don't have the same handler as other ONGOING requests. So let's not create this tech debt.